helpers/notificationHelper.ts

```diff
@@ -43,6 +44,18 @@ export async function createNewReviewOnYourLevelNotification(levelUserId: string
   });
 }
 
+export async function achievementNotification(userIdWhoBeatXAmountOfLevels: string | ObjectId, message?: string | Object){
+  return await NotificationModel.create({
+    source: userIdWhoBeatXAmountOfLevels,
```

Comment:
I think you can do `source: undefined` here, and then you can add a `message: ` based on the achievement type...

But actually just thinking, maybe a better way would be to create the Achievement in the db, then the `source` can be the achievement id. This way when we display the notification we lookup the achievement source, find the type, and display a string based on the type. @k2xl thoughts on this?

Reply:
Are you thinking source: achievement and target: user?

Reply:
Yea I think source could have the achievement id, and target is user id. Then when we get the notification we can lookup the achievement and show different text / have a different icon based on the type

Also allows us to update achievement names/descriptions at any point and the notifications will always display correctly

Reply:
sgtm